1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,90 @@
-// Write your code here
+const arrImages = [
+  "https://cdn.iconscout.com/icon/premium/png-128-thumb/earth-398-995317.png",
+  "https://cdn.iconscout.com/icon/premium/png-128-thumb/saturn-145-995316.png",
+  "https://cdn.iconscout.com/icon/free/png-128/moon-142-215130.png",
+  "https://cdn.iconscout.com/icon/premium/png-128-thumb/venus-41-995329.png",
+  "https://cdn.iconscout.com/icon/premium/png-128-thumb/mercury-43-995314.png",
+  "https://cdn.iconscout.com/icon/premium/png-128-thumb/pluto-21-995310.png",
+  "https://cdn.iconscout.com/icon/premium/png-128-thumb/jupiter-44-995327.png",
+  "https://cdn.iconscout.com/icon/premium/png-128-thumb/mars-45-995318.png",
+];
+
+let image = document.createElement("img");
+let flippingImages;
+let index = 0;
+
+function display() {
+    image.src = arrImages[0];
+    document.getElementById("imagesBox").appendChild(image);
+    // document.getElementById("order").textContent;
+}
+
+document.getElementById("forwardBtn").addEventListener("click", () => {
+  clearInterval(flippingImages);
+    index++;
+    if (index === arrImages.length) index = 0;
+    image.src = arrImages[index];
+    document.getElementById("order").textContent = index;
+});
+
+document.getElementById("backBtn").addEventListener("click", () => {
+  clearInterval(flippingImages); 
+    index--;
```

Comment:
There's a lot of duplicated/repeated code in this solution. Can you extract some of the logic out into its own functions?